src/stdune/path.mli

```diff
@@ -146,3 +146,9 @@ val extension : t -> string
 val drop_prefix : t -> prefix:t -> string option
 
 val pp : Format.formatter -> t -> unit
+
+val build_dir_exists : unit -> bool
+
+val ensure_build_dir_exists : unit -> unit
+
+val relative_build_dir : string -> t
```

Comment:
Maybe `relative_to_build_dir`? `relative_build_dir` makes me think of a `build_dir` constant that is relative